README.md

```diff
@@ -16,10 +16,42 @@ For this, we use an ADR-inspired process.
 
 To make a meaningful contribution:
 
-1. Create an ADR proposal in the [decisions/log](decisions/log/) with status `In Progress`
-    1. If possible, already create the changes to the technical specification text as well (see above)
-2. Create a Pull Request with the ADR proposal and optionally the specification changes
-3. Once there is consensus on the ADR proposal, the ADR proposal is merged into the `main` branch
+## ADR Submission, Review, and Acceptance Process
+Architecture decisions are submitted via creating a ``Markdown``
+
+- ADRs are numbered sequentially and monotonically. Numbers are not reused.
+- Format of the ADR: Please see an [existing ADR](https://github.com/wbcsd/data-exchange-protocol/blob/adr-0021-auth-conformance-issues/decisions/log/0015-pagination.md) for the format
+
+### Submitting an ADR
+
+- Fork and Clone the repo to your local
+- Create a new branch (name as you prefer but recommend to use your GitHub handle + "-" + adr name e.g. <githubhandle>-adr-0016)
+- Create a Markdown file for the ADR under the [decisions/log](decisions/log/) folder in your local branch (based on the proposed format)
+    -  If possible, submit respective changes (based on your ADR proposal) already to the technical specification text as well (see above) 
+- Commit and push the changes to github repo and raise a PR to merge the forked repo into WBCSD `base repo` (for more information please refer [this link](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)). Tag reviewers as necessary or communicate via email to the concerned audience to get the ADR Review process started
```

Comment:
Do we have a process for who should be tagged as reviewers? Not sure how someone raising an ADR would know who to tag as a reviewer.

Reply:
Jus a comment about this: I approved this PR with the `trivets-rhodium` GitHub account (which is just an account I use for quick GH tests) to make sure that any GH user can review PR's in this repo even without being tagged as a reviewer. 
While tagging is nevertheless important, I guess there's also the possibility of asking stakeholders to review a certain PR even without formally tagging them on GH.